docs/playground.md

```diff
@@ -0,0 +1,269 @@
+
+1. Enter your own code snippet or use samples from the Playground.
+2. Optional: Use the `develop` branch docker image by clicking **Tools > Use develop docker image**. This docker image contains the latest code pushed to Semgrep's repository, including experimental features.
+
+3. Press **Run** or **Ctrl+Enter**.
+
+To **use a sample** from the Playground:
+
+1. Click **Tools > Load sample `LANGUAGE` code**.
+![Screenshot of Playground sample code menu](img/playground-sample-code.png "Playground sample code menu")
+2. Select any of the following:
+    1. hello world
+    2. 100-line file
+    3. 1000-line file
+3. The rule is executed automatically on the selected option.
+4. To run again, press Run or type **Ctrl+Enter**.
+
+100-line and 1000-line files are from random from GitHub open source projects.
+
+**Testing a rule** ensures that it meets your standards for precision and speed. Aside from running a rule in the Playground, it is possible to test a rule by creating comment annotations for intended and unintended matches. You can achieve this by creating a test file.
+
+Refer to [Testing rules](../writing-rules/testing-rules) for the syntax and method to run test files.
+
+
+### Debugging errors when creating a rule
+
+Common errors are syntax or match issues.
+
+
+#### The pattern can't be parsed for the language
+
+Check that the **Language is** value and **Test code** language match. Use full AST elements for the language. For example:
+
+* In Java, Python's ``print`` command is not a valid element.
+* ``1+`` or ``if $X:`` are not valid patterns because they are not full AST elements.
+
+#### The pattern syntax is invalid
+
+Check your metavariable definitions, and use of operators. Metavariables must be uppercase letters preceded by a dollar sign $, such as `$PAYMENT_OUTPUT` or `$X`. Refer to [Pattern syntax](../writing-rules/pattern-syntax/).
+
+
+#### The YAML syntax is unparsable in Advanced view
+
+Check for spelling and indentation issues. The key names must match [Semgrep's schema](../writing-rules/rule-syntax/).
+
+#### The rule does not behave as expected or does not find the desired match
+
+File a [bug](https://github.com/returntocorp/semgrep/issues/new?title=semgrep.dev%20bug%20report) or reach out through [Semgrep Community Slack](https://r2c.dev/slack).
+
+
+### Evaluating a rule's performance
+
+After running Semgrep on the test code, a summary of findings appears under the **Matches** column.
+
+The **Performance **column displays how long it takes in seconds for Semgrep to run your rule. It also displays the following benchmarks:
+
+* Blazing
+* Quick
+* Decent
+* Fair
+* Okay
+* Slow
+* Sluggish
+
+These benchmarks take into account the **visual complexity** of the rule.
+
+
+## Adding a rule to the Rule Board
+
+The [Rule Board](../semgrep-app/rule-board/) displays rules that Semgrep App uses to scan your project's code. Adding a rule to the Rule Board allows you to quickly set a rule as part of every Semgrep scan.
```

Comment:
I have noticed this description in the old newsletter. So this means we can drag and drop things etc.? Maybe we can mention it. So Rule Board is essentially a live kanban board for rules? :-) @s-santillan 
![Screenshot 2022-02-04 at 13 36 42](https://user-images.githubusercontent.com/26669307/152530202-dfd226d7-d5e6-4c67-b54d-b45c69f1bbec.jpg)

Reply:
Updated description in latest commit! :) Thanks for the catch.